config_task_limits.go

```diff
@@ -36,6 +36,9 @@ type TaskLimitsConfig struct {
 	// MaxConcurrentLargeParserProjectTasks is the maximum number of tasks with >16MB parser projects that can be running at once.
 	MaxConcurrentLargeParserProjectTasks int `bson:"max_concurrent_large_parser_project_tasks" json:"max_concurrent_large_parser_project_tasks" yaml:"max_concurrent_large_parser_project_tasks"`
 
+	// MaxDegradedModeConcurrentLargeParserProjectTasks is the maximum number of tasks with >16MB parser projects that can be running at once during degraded mode.
```

Comment:
`during CPU degraded mode` to make it consistent with the comment on `MaxDegradedModeParserProjectSize`.